packages/url_launcher/url_launcher_ios/example/ios/RunnerTests/URLLauncherTests.swift

```diff
@@ -123,31 +120,34 @@ final class URLLauncherTests: XCTestCase {
     let plugin = createPlugin(launcher: launcher)
 
     let expectation = XCTestExpectation(description: "completion called")
-
-    plugin.launchURL("good://url", universalLinksOnly: true) { result, error in
-      XCTAssertNil(error)
+    plugin.launchUrl(url: "good://url", universalLinksOnly: true) { result in
+      switch result {
+      case .success(let details):
+        XCTAssertEqual(details.result, .success)
+      case .failure(let error):
+        XCTFail("Unexpected error: \(error)")
+      }
       expectation.fulfill()
     }
 
     wait(for: [expectation], timeout: 1)
-
     XCTAssertEqual(launcher.passedOptions?[.universalLinksOnly] as? Bool, true)
   }
 
 }
 
-final private class FakeLauncher: NSObject, FULLauncher {
+final private class FakeLauncher: NSObject, Launcher {
   var passedOptions: [UIApplication.OpenExternalURLOptionsKey: Any]?
 
-  func canOpen(_ url: URL) -> Bool {
-    return url.scheme == "good"
+  func canOpenURL(_ url: URL) -> Bool {
+    url.scheme == "good"
   }
 
-  func open(
-    _ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any] = [:],
-    completionHandler: ((Bool) -> Void)? = nil
+  func openURL(
+    _ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any],
+    completionHandler completion: ((Bool) -> Void)?
```

Comment:
curious if this line wrap is done by swift-formatter? it's odd to layout 2 params in 1 line and another in a separate line. swift developers typically write: 

```
func openURL(
  _ url: URL,
  options: [UIAppli..., 
  completionHandler...
)
```

Reply:
Just looked up [our style guide](https://google.github.io/swift/#line-wrapping): 

> Comma-delimited lists are only laid out in one direction: horizontally or vertically. In other words, all elements must fit on the same line, or each element must be on its own line. 

Can you update the format for this function and a few others as well?